tests/test_processors.py

```diff
@@ -321,3 +331,76 @@ def aggregateRating(self):
     assert page.aggregateRating == AggregateRating(
         ratingValue=3.8, bestRating=10, reviewCount=5
     )
+
+
+@pytest.mark.parametrize(
+    "input_value,expected_value",
+    [
+        (None, None),
+        ([], []),
+        ("https://www.url.com/img.jpg", [Image(url="https://www.url.com/img.jpg")]),
+        (
+            [
+                Image("https://www.url.com/img1.jpg"),
+                Image("https://www.url.com/img2.jpg"),
+            ],
+            [
+                Image("https://www.url.com/img1.jpg"),
+                Image("https://www.url.com/img2.jpg"),
+            ],
+        ),
+        (
+            ["https://www.url.com/img1.jpg", "https://www.url.com/img2.jpg"],
+            [
+                Image("https://www.url.com/img1.jpg"),
+                Image("https://www.url.com/img2.jpg"),
+            ],
+        ),
+        (
+            [
+                {"url": "https://www.url.com/img1.jpg"},
+                {"url": "https://www.url.com/img2.jpg"},
+            ],
+            [
+                Image("https://www.url.com/img1.jpg"),
+                Image("https://www.url.com/img2.jpg"),
+            ],
+        ),
+    ],
+)
+def test_images(input_value, expected_value):
+    class ImagesPage(BasePage):
+        @field(out=[images_processor])
```

Comment:
Does it work without `out`, as images_processor is enabled by default in this PR? It'd be nice to test it without `out`, because it's going to be the most important use case (default behavior).

Reply:
These are tests for processors, not particular pages, which is why I think it makes sense to limit to `BasePage`. Copied this approach from other test in this file, such as [test for Gtin processor](https://github.com/zytedata/zyte-common-items/blob/main/tests/test_processors.py#L221-L228). After all, same processor can be applied to many different page types.